### ISSUE TITLE
Patterns don't work in tolgee.config.ts on Windows

### ISSUE THREAD

**Issue by Chrissi2812** [tolgee/tolgee-cli#130]
`npx tolgee extract print` does not find any files when I use the `patterns` option in the config file.
I have to override it via `--patterns`, only then it finds files.

I tried it in GitBash and CMD both returned the same result. When using the Patterns inside config file they get transformed to absolute paths that don't seem to work.

```bash
npx tolgee extract print
[ 'E:\\github\\_RETRACTED_\\pages\\**\\*.{ts,tsx,vue}' ]
🔴 No files were matched for extraction


npx tolgee extract print --patterns ./pages/**/*.{ts,tsx,vue}
[ './pages/**/*.{ts,tsx,vue}' ]
🐭✅     Analyzing code...
5 keys found in pages\login.vue:
        line 25: email
        line 35: password
        line 40: forgot_password
        line 50: dont_have_account
        line 52: register

Total unique keys found: 5
Total warnings: 0
```

The config file in use
```ts
export default {
  projectId: 123,
  format: 'JSON_TOLGEE',
  patterns: [
    // './lib/**/*.{ts,tsx,vue}',
    './pages/**/*.{ts,tsx,vue}',
    // './components/**/*.{ts,tsx,vue}',
    // './app/**/*.{ts,tsx,vue}',
    // './src/**/*.{ts,tsx,vue}',
    // './{A,a}pp.{vue,js,jsx,mjs,ts,tsx}'
  ],
  parser: 'vue',
  push: {
    files: [
      {
        path: './i18n/locales/en-US.json',
        language: 'en-US',
      },
      {
        path: './i18n/locales/de-DE.json',
        language: 'de-DE',
      }
    ]
  },
  pull: {
    path: './i18n/locales',
  }
}
```

**Comment by cyyynthia** [comment]
May be resolved by #129 🤔

**Comment by cyyynthia** [comment]
Can you confirm this issue is fixed as of 2.4.0? 🙏🏻

**Comment by Chrissi2812** [comment]
Yes everything works now 🎉
